node/src/neighborhood/mod.rs

```diff
@@ -3898,99 +4004,74 @@ mod tests {
                     FallbackPreference::ExitCountryWithFallback
                 );
                 assert_eq!(
-                    neighborhood
-                        .neighborhood_database
-                        .node_by_key(&es_public_key)
-                        .unwrap()
-                        .metadata
-                        .country_undesirability,
+                    neighborhood.get_node_country_undesirability(&es_public_key),
                     UNREACHABLE_COUNTRY_PENALTY,
-                    "es We expecting {}, country is too close to be exit",
+                    "es We expect {}, country is too close to be exit",
```

Comment:
Tady by se to dalo vylepsit podobne jako naznaceno uz v tom predeslem komentari...ale je to spis jen me prani...